gammapy/scripts/info.py

```diff
@@ -1,105 +1,83 @@
-        _print_info_dependencies()
-
-
-def _print_info_version():
-    """Print Gammapy version info."""
-    from gammapy import version
-    print('\n*** Gammapy version info ***\n')
-    print('version: {}'.format(version.version))
-    print('release: {}'.format(version.release))
-    print('githash: {}'.format(version.githash))
-    print('')
-
-
-def _print_info_tools():
-    """Print info about Gammapy command line tools."""
-    print('\n*** Gammapy tools ***\n')
-
-    # TODO: how to get a one-line description or
-    # full help text from the docstring or ArgumentParser?
-    # This is the function names, we want the command-line names
-    # that are defined in setup.py !???
-    from gammapy.utils.scripts import get_all_main_functions
-    scripts = get_all_main_functions()
-    names = sorted(scripts.keys())
-    for name in names:
-        print(name)
-
-    # Old stuff that doesn't work any more ...
-    # We assume all tools are installed in the same folder as this script
-    # and their names start with "gammapy-".
-    # import os
-    # from glob import glob
-    # DIR = os.path.dirname(__file__)
-    # os.chdir(DIR)
-    # tools = glob('gammapy-*')
-    # for tool in tools:
-    #     # Extract first line from docstring as description
-    #     description = 'no description available'
-    #     lines = open(tool).readlines()
-    #     for line in lines:
-    #         if line.startswith('"""'):
-    #             description = line.strip()[3:]
-    #             if description.endswith('"""'):
-    #                 description = description[:-3]
-    #             break
-    #     print('{0:35s} : {1}'.format(tool, description))
-
-    print('')
-
-
-def _print_info_dependencies():
-    """Print info about Gammapy dependencies."""
-    print('\n*** Gammapy dependencies ***\n')
-    from gammapy.conftest import PYTEST_HEADER_MODULES
+    if no_args:
+        print(parser.description)
+
+
+def print_info(info, title):
+    info_all = '\n{title}:\n\n'.format(title=title)
+
+    for key in sorted(info):
+        info_all += '\t{key:22s} : {value:<10s} \n'.format(key=key, value=info[key])
 
+    print(info_all)
+
+
+def get_info_version():
+    info_version = {}
+    info_version['version'] = version.version
+    info_version['release'] = str(version.release)
+    info_version['githash'] = version.githash
+    return info_version
+
+
+def get_info_dependencies():
+    """Get info about Gammapy dependencies."""
+
+    info_dependencies = {}
     for label, name in PYTEST_HEADER_MODULES.items():
         try:
             module = importlib.import_module(name)
             version = module.__version__
         except ImportError:
             version = 'not available'
+        info_dependencies[label] = version
+    return info_dependencies
+
+
+def get_info_system():
+    """Print info about Gammapy dependencies."""
+    info_system = {}
+
+    env_variables = ['GAMMAPY_EXTRA', 'HGPS_DATA', 'GAMMAPY_FERMI_LAT_DATA',
```

Comment:
Move this list of env variables out of the function?
We might want to access it from somewhere else, e.g. to add to it, and that's not possible if it's inside `get_info_system`.